addons/portal_rating/static/src/xml/portal_chatter.xml

```diff
@@ -72,7 +72,7 @@
 
     <t t-name="portal_rating.chatter_rating_publisher_comment">
         <div class="d-flex o_portal_chatter_message">
-            <img class="o_portal_chatter_avatar" t-att-src="rating.publisher_avatar" alt="avatar"/>
+            <img class="o_portal_chatter_avatar o_avatar" t-att-src="rating.publisher_avatar" alt="avatar"/>
```

Comment:
The `o_portal_chatter_avatar` class seems more specific than the `o_avatar` class so maybe using this order is a bit better: `o_avatar o_portal_chatter_avatar`